apps/src/templates/policy_compliance/ChildAccountConsent/index.scss

```diff
@@ -0,0 +1,41 @@
+@import 'color';
+@import 'font';
+
+#permission_granted_container, #expired_token_container {
+  border: 1px solid rgb(233 233 233);
+  box-shadow: 5px 5px 3px 0 #ccc;
+  box-sizing: border-box;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 10px;
+  max-width: 700px;
+  padding: 36px;
+
+  & h4 {
+    color: #00adbc;
+  }
+
+  & p {
+    margin-bottom: 20px;
+  }
+
+  & p.permission-granted-date {
+    margin: 28px 0;
+
+    & .title {
+      display: inline-block;
+      color: $light_gray_800;
+      background-color: rgb(255 214 0 / 0.25);
```

Comment:
@alivira was this background color for the title "Permission" intentional? I see it in the figma but thought it might have been an accidental highlight.

Reply:
Ah no not intentional - that must have been on there for a screenshot at some point